### PR TITLE
JHEF722HD: Fix some definition deficiencies, re-enable barometer and PIN2

### PR DESCRIPTION
**Pull request by jhemcu**

### CODE REVIEW

File: configs/JHEF722HD/config.h [hunk 1]
```diff
@@ -28,11 +28,11 @@
...
 #define USE_ACC
 #define USE_ACC_SPI_MPU6000
-#define USE_ACC_SPI_ICM42688P   
+#define USE_ACCGYRO_BMI270
+#define USE_ACC_SPI_ICM42688P
 #define USE_GYRO
 #define USE_GYRO_SPI_MPU6000
 #define USE_GYRO_SPI_ICM42688P
-#define USE_ACCGYRO_BMI270
 #define USE_BARO_DPS310
```
Comment (haslinghuis) [inline configs/JHEF722HD/config.h:36]:
```suggestion
#define USE_BARO
#define USE_BARO_DPS310
```

File: configs/JHEF722HD/config.h [hunk 2]
```diff
@@ -96,26 +97,26 @@
 
 #define ADC3_DMA_OPT 0
 
-#define MAG_I2C_INSTANCE  (I2CDEV_1)
+#define DEFAULT_GYRO_TO_USE GYRO_CONFIG_USE_GYRO_BOTH
+#define MAG_I2C_INSTANCE (I2CDEV_1)
+#define USE_BARO
```
Comment (haslinghuis) [inline configs/JHEF722HD/config.h:102]:
```suggestion

```

File: configs/JHEF722HD/config.h [hunk 3]
```diff
@@ -96,26 +97,26 @@
...
 #define PINIO1_BOX 40
+#define PINIO2_BOX 41
+#define FLASH_SPI_INSTANCE SPI3
+#define GYRO_1_SPI_INSTANCE SPI1
 #define GYRO_1_ALIGN CW270_DEG
 #define GYRO_1_ALIGN_YAW 2700
+#define GYRO_2_SPI_INSTANCE SPI1
 #define GYRO_2_ALIGN CW270_DEG
-#define GYRO_2_ALIGN_YAW 2700
+#define GYRO_2_ALIGN_YAW 2700
```
Comment (haslinghuis) [inline configs/JHEF722HD/config.h:122]:
```suggestion
#define GYRO_2_ALIGN_YAW 2700

```

**Comment by jhemcu** [comment]
Rebased it~!

File: configs/JHEF722HD/config.h [hunk 4]
```diff
@@ -94,28 +96,26 @@
...
-#define BARO_I2C_INSTANCE (I2CDEV_1)
-
-#define FLASH_SPI_INSTANCE   SPI3
-#define GYRO_1_SPI_INSTANCE  SPI1
-#define GYRO_2_SPI_INSTANCE  SPI1
-#define MAX7456_SPI_INSTANCE SPI2
-
-#define SERIALRX_PROVIDER SERIALRX_CRSF
+#define ADC3_DMA_OPT                    0
+#define DEFAULT_GYRO_TO_USE             GYRO_CONFIG_USE_GYRO_BOTH
```
Comment (haslinghuis) [inline configs/JHEF722HD/config.h:100]: Board has two equal gyro's?

**Review by nerdCopter — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by nerdCopter** [comment]
no `UART3_TX_PIN` correct?

**Comment by haslinghuis** [comment]
@jhemcu any reference (wire diagram) or documentation?

**Comment by haslinghuis** [comment]
Guessing here: 

image

https://ymzfpv.com/products/jhemcu-ghf722-f7-hd-2-6s-aio-whoop-toothpick-flight-controller-w-40a-16bit-4in1-esc?variant=47658431414591

**Comment by ot0tot** [comment]
> Guessing here:
> 
> image
> 
> https://ymzfpv.com/products/jhemcu-ghf722-f7-hd-2-6s-aio-whoop-toothpick-flight-controller-w-40a-16bit-4in1-esc?variant=47658431414591

image

**Comment by haslinghuis** [comment]
Yes no idea about original board documentation. Will be mandatory for future PRs.

**Comment by jhemcu** [comment]
In order not to cause misunderstanding, even though it has been merged, I want to clarify it here.

> no `UART3_TX_PIN` correct?

Yes, this board does not lead to the serial 3 TX pin, because only the serial 3 RX pin is needed.

image



> Guessing here:
> 
> image
> 
> https://ymzfpv.com/products/jhemcu-ghf722-f7-hd-2-6s-aio-whoop-toothpick-flight-controller-w-40a-16bit-4in1-esc?variant=47658431414591

This is another product of our company, so don’t get confused.


@nerdCopter  @haslinghuis 


Thank you again for your support and help to us~

**Pull request merged**